dvc/command/plots.py

```diff
@@ -138,10 +147,10 @@ def add_parser(subparsers, parent_parser):
         help="Required when CSV or TSV datafile does not have a header.",
     )
     plots_show_parser.add_argument(
-        "--show-json",
+        "--show-vega",
         action="store_true",
         default=False,
-        help="Show output in JSON format.",
+        help="Show output in VEGA format.",
```

Comment:
```suggestion
        help="Show output in Vega format.",
```

Reply:
Same in line 216

Reply:
Actually, these are fixed. Probably did that after.

Reply:
I'm seeing this in the output of `dvc plots show -h` in `1.0.0a7+12172f` which I think is the latest `master` version.

Reply:
@jorgeorpinel Oh, sorry, I'm blind. I looked at the bottom part that is the patch itself and not your suggestion. Will fix, thank you!